### ISSUE TITLE
convert the mode to onnx,but BatchNorm2d is lost

### ISSUE THREAD

**Issue by cqray1990** [onnx/onnx#4829]
# Bug Report

### Describe the bug

                          class Net(nn.Module):
                              def __init__(self):
                                  super(Net, self).__init__()
                                  self.conv = nn.Conv2d(
                                      3,
                                      64,
                                      3,
                                      stride=2,
                                      padding=0,
                                  )
                                
                                  self.bn = nn.BatchNorm2d(64)
                                  self.relu = nn.ReLU(inplace=True)
                          
                              def forward(self,inx):
                                  x = self.conv(inx)
                                  x = self.bn(x)
                                  x = self.relu(x)
                          
                                  return x
                          
                          
                          model = Net()
                          model = model.cuda()
                          
                          x = torch.rand([1, 3, 224, 224]).to("cuda")
                          out = model(x)
                          print(out)
                          print(model)
                          torch.onnx.export(model,
                                                    x,
                                                    "./modelr.onnx",
                                                    export_params=True,
                                                    verbose=True,
                                                    opset_version=11,
                                                    do_constant_folding=True,
                                                    input_names=["input"], output_names=["output"],
                                                    )
                          
                          
                          
                          netron.start("./modelr.onnx")
![modelr onnx](https://user-images.githubusercontent.com/32585434/215685466-3163ca21-cdfa-4667-b1bb-e341f50135c2.png)

**Comment by jcwchen** [comment]
Hi @cqray1990,
It should be a question for the exporter. I saw you also raise this issue in the exporter repo (under PyTorch), which is a better place to track this issue. I will close this one. Feel free to reopen it if you still have other ONNX questions. Thanks.
